WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -105,20 +105,20 @@
 
 <script>
   <% if @past_selected
-    state_click = "past"
-  elsif @present_selected
-    state_click = "present"
-  elsif @recent_selected
-    state_click = "recent"
-    end %>
+       state_click = "past"
+     elsif @present_selected
+       state_click = "present"
+     elsif @recent_selected
+       state_click = "recent"
+     end %>
```

Comment:
Actually it could be
```erb
  <% state_click = if @past_selected
                     "past"
                   elsif @present_selected
                     "present"
                   elsif @recent_selected
                     "recent"
                   end %>
```
It'd be even better if we'd sanitize `params[:state]` to have one of these values as discussed [here](https://github.com/cubing/worldcubeassociation.org/issues/893).